extensions.json

```diff
@@ -1163,8 +1159,6 @@
     {
       "id": "nwolverson.ide-purescript",
       "repository": "https://github.com/nwolverson/vscode-ide-purescript",
-      "version": "0.23.3",
-      "checkout": "v0.23.3",
```

Comment:
Well okay, the latest release wasn't tagged, so let's just publish from the default branch now.